packages/block-library/src/buttons/save.js

```diff
@@ -3,9 +3,14 @@
  */
 import { InnerBlocks } from '@wordpress/block-editor';
 
-export default function save() {
+export default function save( { attributes } ) {
+	const { blockDirection } = attributes;
 	return (
-		<div>
+		<div
+			className={
+				blockDirection === 'vertical' ? 'is-direction-vertical' : ''
```

Comment:
Same thing here with the suggestion to use `classnames`.

Reply:
Done, thank you! ❤️